DOC_CHANGES.md

```diff
@@ -78,3 +78,42 @@ either reboot immediately (:reboot_now) or queue a reboot (:request_reboot).  Th
 
 The --identity-file option to `knife bootstrap` has been deprecated in favor of `knife bootstrap --ssh-identity-file`
 to better align with other ssh related options.
+
+### `windows_package` resource
+
+`windows_package` now supports more than just `MSI`. Most common windows installer types are supported including inno setup, nullsoft, wise and installshield. The new allowed `installer_type` values are: `inno`, `nsis`, `wise`, `installshield`, `custom`, and `msi`.
+
+Also, while being able to download remote installers from a `HTTP` resource is not new, it looks as though the top of the docs page is incorrect stating that only local installers can be used as a source.
```

Comment:
Hmm, I have assumed in my earlier edits that this file is for letting people know what docs have changed.